test/test_commit.py

```diff
@@ -149,6 +149,81 @@ def check_entries(d):
         self.assertEqual(commit.committer_tz_offset, 14400, commit.committer_tz_offset)
         self.assertEqual(commit.message, "initial project\n")
 
+    def test_patch_with_parent(self):
```

Comment:
Since this is a patch with parent, I think it should be easy to find a patch with is much smaller. If there truly is no way, the expected value could be read from a file.

Reply:
Thanks for reviewing it. I will try to get commit with a smaller patch.